spring-cloud-sleuth-core/src/main/java/org/springframework/cloud/sleuth/Log.java

```diff
@@ -30,7 +33,7 @@
 	private final long timestamp;
 
 	/**
-	 * Event (if not null) should be the stable name of some notable moment in the lifetime of a span.
+	 * Event should be the stable name of some notable moment in the lifetime of a span.
```

Comment:
This was only allowed to be null in OpenTracing as the object was a union type. (Ex there were other fields which might be missing, but one had to be present).